CMakeLists.txt

```diff
@@ -193,6 +193,10 @@ option_with_default(BUILD_Libint2_GENERATOR "If prebuilt not found, don't build
 option_with_default(psi4_SHGAUSS_ORDERING "Ordering for shells of solid harmonic Gaussians: standard -- standard ordering (-l, -l+1 ... l) -or- gaussian -- the Gaussian ordering (0, 1, -1, 2, -2, ... l, -l). Changing this is experimental, expert only, and not a future plan. Internal calculations should be correct and fchk files should be correct, but other exports like Molden and QCSchema.Wavefunction won't be." gaussian)
 option_with_default(gauxc_ENABLE_GPU "Enable GPU support for GauXC" OFF)
 option_with_default(gauxc_ENABLE_MAGMA "Enable MAGMA support for GauXC. Required GPU-enabled GauXC" OFF)
+# sadly, it seems FORCE_PEDANTIC runs into issues with GauXC when option_with_flag is used
+# option_with_default + setting -pedantic-errors in psi4/CMakeLists.txt seems to fix this
+option_with_default(FORCE_PEDANTIC "Forces code to comply with the language standard." OFF)
```

Comment:
if this works you could try switching to `option_with_print`, which is the normal "bool" one (not that one could guess that from the immediate vicinity). `-pedantic-errors` is really only targeting one compiler family, so I'd say this is find as not taking full advantage of `option_with_flags` anyway.

Reply:
Got it! I'll test that locally at least.

Reply:
Decided to switch to `option_with_print` since that seems to work locally as well.